api/src/main/java/org/pmiops/workbench/db/model/DbUserRecentlyModifiedResource.java

```diff
@@ -0,0 +1,102 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "user_recently_modified_resource")
+public class DbUserRecentlyModifiedResource {
```

Comment:
this version does not provide `getCohort()` and `getConceptSet()` - do you have a plan for how to replace this functionality?

Reply:
I am planning to handle that in the retrieval side of PR. This PR is just for maintaining a true copy of database copy from user_recent_resource

Reply:
ok